src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Dto/LearnerData.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Dto
+{
+    public class LearnerData
+    {
+        public LearnerData(string learnerRefNumber)
+        {
+            LearnRefNumber = learnerRefNumber;
+        }
+        public string LearnRefNumber { get; }
+        public List<RequiredPaymentEntity> RequiredRefunds { get; } = new List<RequiredPaymentEntity>();
```

Comment:
I propose a shared kernel for shared domain entities
Initially I would put a RequiredPayment (which is now a domain entity) in there and use one of those here.

No reason to duplicate the code

Reply:
Agree to disagree :) if we have time I'd like to create it. But not now

Reply:
Agreed (or disagreed)